examples/animation/animate_decay.py

```diff
@@ -32,6 +32,6 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
+ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
```

Comment:
As the animation goes on, the limits keep changing. When blitting is turned
on, you don't see the x-axis change, which is incorrect. Turn blitting off,
and you see the scale change correctly.

On Thu, Jan 8, 2015 at 10:12 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In examples/animation/animate_decay.py
> <https://github.com/matplotlib/matplotlib/pull/3985#discussion-diff-22699612>
> :
>
> > @@ -32,6 +32,6 @@ def run(data):
> >
> >      return line,
> >
> > -ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
> > +ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
>
> why turn of
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3985/files#r22699612>.
>